tools/db_crashtest.py

```diff
@@ -115,8 +115,8 @@
     "use_direct_reads": lambda: random.randint(0, 1),
     "use_direct_io_for_flush_and_compaction": lambda: random.randint(0, 1),
     "mock_direct_io": False,
-    "cache_type": "lru_cache",  # clock_cache is broken
-                                # fast_lru_cache is currently incompatible with stress tests, because they use strict_capacity_limit = false
+    "cache_type": "lru_cache",  # fast_lru_cache and clock_cache are currently incompatible
```

Comment:
Yes, it's not broken any more.

Reply:
Note that #10262 is starting the path toward making them compatible, because insertion might fail because of fixed table size even without strict_capacity_limit. We might also consider returning a dynamic Handle to pretend that insertion succeeds even when table is full. (For that, Release would need to be able to detect the other kind of Handle.)